base_running/app/midi_controller.py

```
@@ -0,0 +1,40 @@
+# --- 의존성 주입 ---
+def get_midi_service() -> midi_service.MidiService:
+    return midi_service.midi_service
+
+# --- API 라우터 ---
+router = APIRouter(
+    prefix="/midi",
+    tags=["MIDI Generation"],
+)
+
+@router.post("/generate")
+async def generate_midi_endpoint(request: MidiRequest, svc: midi_service.MidiService = Depends(get_midi_service)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)